### PR TITLE
Fixed: The report system needs to be checked and agreed upon

### PR DESCRIPTION
**Pull request by zackjohnson8**

### CODE REVIEW

File: app/post/views.py [hunk 1]
```diff
@@ -398,38 +398,48 @@ def _createreport(postid):
...
-    current_user = get_jwt_identity();
-    if current_user:
-        user = User.query.filter_by(username=current_user).first()
-        is_staff = user.is_staff
-        if is_staff:
-            reports = Report.query.filter_by(reported_post_id=postid).all()
-            result = reports_schema.dump(reports, many=True)
-            return jsonify({'reports': result})
-    return jsonify('unauthorized'), 403
+    # current_user = get_jwt_identity()
```
Comment (NathanYocum) [inline app/post/views.py]: We should still have post reporting and general error reporting

Comment (zackjohnson8) [reply app/post/views.py]: Alright so we'll do both. I'll make sure we have both options then before merging this. Working on fixing the bugs I've encountered with this merge.

File: app/post/views.py [hunk 2]
```diff
@@ -412,43 +424,98 @@ def _createreport(postid):
...
         report = Report(
             description=json_data['description'],
             reported_content_extension=json_data['extension']
         )
-    return jsonify({"message": "unauthorized"}), 403
+        return jsonify(message="success"), 200
+    return jsonify(message="unauthorized"), 403
+
+
+@postblueprint.route('/v1/report/createReportPost/<int:postid>', methods=['POST'])
```
Comment (NathanYocum) [inline app/post/views.py]: Route should end with / for consistency

Comment (zackjohnson8) [reply app/post/views.py]: Oh good call I didn't notice that.

**Pull request merged**